ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-main-postsubmits.yaml

```diff
@@ -14,9 +14,9 @@ postsubmits:
       containers:
       - args:
         - --config-dir=../../openshift/release/ci-operator/config/open-cluster-management/clusterlifecycle-state-metrics/
-        - --current-release=2.4
```

Comment:
suggest use ocm-ci-fastforward https://github.com/openshift/release/blob/19718ba06831c899f80e9a563403ff5cfa72f0ca/ci-operator/config/open-cluster-management/helloprow-go/open-cluster-management-helloprow-go-main.yaml#L104 to do ff

Reply:
@zhiweiyin318 thx, fixed.